pkg/nmi/nmi.go

```diff
@@ -0,0 +1,28 @@
+package nmi
+
+import (
+	"context"
+
+	aadpodid "github.com/Azure/aad-pod-identity/pkg/apis/aadpodidentity"
+	"github.com/Azure/go-autorest/autorest/adal"
+)
+
+// OperationMode is the mode in which NMI is operating
+// allowed values: standard, managed
+type OperationMode string
+
+const (
+	// StandardMode ...
+	StandardMode OperationMode = "standard"
+	// ManagedMode ...
+	ManagedMode OperationMode = "managed"
+)
+
+// TokenClient ...
+type TokenClient interface {
+	// GetIdentities gets the list of identities which match the
```

Comment:
Can we please add a README and link it to the features list which explains the token interface and the split. Some short text there would be great to help newer developers get more clarity on what is going on.

Reply:
will add readme before release like discussed during code review.